Add CLI examples for CloudWatch Logs (#8995)
aws/aws-cli: awscli/examples/logs/put-query-definition.rst

> **To Create or update a query definition for CloudWatch Logs Insights**

Reviewer: ```suggestion
**To create or update a query definition for CloudWatch Logs Insights**
```

> The following ``put-query-definition`` example creates a query definition for CloudWatch Logs Insights. If the command succeeds, no output is returned. ::

Reviewer: ```suggestion
The following ``put-query-definition`` example creates a query definition for CloudWatch Logs Insights. ::
```

    aws logs put-query-definition \
        --name DemoLogsQueries/Example \
        --log-group-names arn:aws:logs:us-east-1:123456789012:log-group:demo-log-group \

>         --query-string "stats sum(packets) as packetsTransferred by srcAddr, dstAddr | sort packetsTransferred desc | limit 100"

Reviewer: ```suggestion
        --query-string 'stats sum(packets) as packetsTransferred by srcAddr, dstAddr | sort packetsTransferred desc | limit 100'
```
